glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt

```diff
@@ -394,6 +394,16 @@ open class GleanInternalAPI internal constructor() {
         return gleanSetSourceTags(tags.toList())
     }
 
+    /**
+     * Set configuration for metrics' disabled property, typically from a remote_settings
+     * experiment or rollout.
+     *
+     * @param json Stringified JSON map of metrics and their associated `disabled` property.
+     */
+    fun setMetricsDisabledConfig(json: String) {
+        gleanSetMetricsDisabledConfig(json)
```

Comment:
@travis79 What type does the config have over in Nimbus Kotlin/Swift? Does it make sense to just take the string here or is it possible to take a dedicated object (or json object?)

Reply:
@badboy 
I chose to make it a string since we already check that the JSON is actually a `MetricsDisabledConfig` type.
https://github.com/mozilla/glean/blob/main/glean-core/src/lib.rs#L778

Reply:
oh perfect. Sorry for the run-around then.